File: app/src/main/java/com/amarnehsoft/holyquran/main/SplashActivity.java
package com.amarnehsoft.holyquran.main;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        startActivity(MainActivity.newIntent(this));
        finish();
    }
}

File: app/src/main/java/com/amarnehsoft/holyquran/main/SplashActivity.kt
package com.amarnehsoft.holyquran.main

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.amarnehsoft.holyquran.ui.DownloadQuranActivity
import com.amarnehsoft.holyquran.ui.DownloadQuranViewModel

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        startActivity(DownloadQuranActivity.newIntent(this))
        finish()
    }
}
